src/utils/index.js

```diff
@@ -193,6 +193,25 @@ export default {
     return true
   },
 
+  regexCompare (value, search) {
+    try {
+      const regexpParts = search.match(/^\/(.*?)\/([gim]*)$/)
+      let regex
+
+      if (regexpParts) {
+        regex = new RegExp(regexpParts[1], regexpParts[2])
+      } else {
+        regex = new RegExp(search, 'gim')
+      }
+
+      if (value.toString().search(regex) !== -1) {
+        return true
+      }
+    } catch (e) {
```

Comment:
Which kind of exceptions can we have here? Is it worth to throw it and display it in the dev console?

Reply:
As example `[a` is no valid regex as the square bracket must be closed (`[a]`).
So it is possible that we run in the catch if the user don't write quickly, so its fine that we dont display it in the dev console.
Also its no critical error i guess.